proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,315 @@
+# AWS CLI Contributing Guide
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
+
+## Abstract
+
+This document proposes standards and processes to improve and publish community
+expectations for contributions. It defines the process for proposing,
+submitting, and reviewing a contribution. This proposal will make it easier for
+the community to successfully contribute to the AWS CLI.
+
+## Motivation
```

Comment:
For the `Motivation` section, let's make the following updates:

1. Rename it to `Background/Motivation`
2. Add a more detailed explanation of the current process as the first section
3. Make the examples section the next section and make sure it focuses on issues with the current process described in the previous section
4. Make the goals section the last section

These changes should help the entire background/motivation section flow a bit better because it will help build up context so that it is easier to understand why we got to the goals we propose by first explaining what the current state of the world is and then saying what the problems are with it.

Reply:
Done.